docker-compose.yml

```diff
@@ -473,22 +539,19 @@ services:
   #
   faf-aio-replayserver:
     container_name: faf-aio-replayserver
-    image: faforever/faf-aio-replayserver:0.8.3
+    # build: git://github.com/FAForever/faf-aio-replayserver#0.1
+    image: faforever/faf-aio-replayserver:0.4.11
```

Comment:
I'm still thinking, that you're reverting mazornoob changes @ replay server

Reply:
fixed